bibtex-actions.el

```diff
@@ -585,8 +624,7 @@ FORMAT-STRING."
 
 (defun bibtex-actions-citation-key-at-point ()
```

Comment:
Is this supposed to be the single key at point or else a list of contiguous keys in a citation command? I went for the latter in latex but both the org one and the previous `bibtex-completion` implementation would return a single key.